modules/auth-schema.js

```diff
@@ -0,0 +1,34 @@
+import Oauth2Scheme from '@nuxtjs/auth-next/dist/schemes/oauth2'
+import { getResponseProp } from '@nuxtjs/auth-next/dist/utils'
+
+import { camelCaseDeep } from '~/utility/case'
+
+export default class System76Schema extends Oauth2Scheme {
+  async fetchUser () {
+    if (!this.check().valid) {
+      return
+    }
+
+    if (!this.options.endpoints.userInfo) {
+      this.$auth.setUser({})
+      return
+    }
+
+    const response = await this.$auth.requestWith(this.name, {
+      url: this.options.endpoints.userInfo
+    })
+
+    const body = getResponseProp(response, this.options.user.property)
+    const user = camelCaseDeep(body)
```

Comment:
The only thing in this file different from the normal nuxt implementation is camelcasing the user record here, and a little bit of the logout redirect logic.

I'm going to try opening a PR to nuxt auth to add hooks, so we can just hook in to rewrite this stuff instead of having to extend the whole schema.